images/build-libs/Makefile

```diff
@@ -30,3 +34,14 @@ push-image: create-builder
 
 local-registry: create-builder
 	docker buildx build --push --platform $(ARCHS) -t "localhost:5000/$(IMG_NAME)" .
+
+sign-image: cosign
+	$(cosign) sign --key $(cosign_key) $(IMAGE)
+
+cosign: go
```

Comment:
maybe we should replace this target name with "install-cosign," wdyt?

Reply:
I think that as Cosign is a tool, is a tool target to be satisfied as a dependency, but it's only about names :)